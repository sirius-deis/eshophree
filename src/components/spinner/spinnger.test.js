import { render, screen } from "@testing-library/react";
import Spinner from "./spinner";

describe("Spinner component", function () {
  it("should match snapshot", function () {
    const { container } = render(<Spinner />);
    expect(container).toMatchSnapshot();
  });
})