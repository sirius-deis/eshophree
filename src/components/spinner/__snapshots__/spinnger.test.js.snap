// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component should match snapshot 1`] = `
<div>
  <div
    class="sc-bgqQcB ABTdF"
  >
    <div
      class="sc-gTRrQi chearB"
    />
  </div>
</div>
`;
